CONTRIBUTING.md

```diff
@@ -70,6 +70,27 @@ manager, you will need to find and install the following tools:
 - JDK 8
 - `sbt`
 - `z3`
+- `gmp`
+- `mpfr`
+
+After installing the tools listed above, you need to download Apron source code,
```

Comment:
Actually if there are linux versus mac specific parts, then put those sections in the Mac/Linux installation sections, respectively